openapi.yaml

```diff
@@ -15129,7 +15129,7 @@ components:
             last_successful:
               type: string
```

Comment:
hey @hzoppetti, I'm wondering what your opinion on adding `nullable: true` to this property might be. I'm a little undecided about it, because my reading of the `nullable` documentation from Swagger is that it might be used more on non-readonly properties; it would communicate that the API user can send `null` for the property:

https://swagger.io/docs/specification/data-models/data-types/#null
https://swagger.io/specification/#schemaNullable

Reply:
right - it is technically nullable - but only by the system, not by users...

The openapi spec says "Allows sending a null value for the defined schema. Default value is false." so I vote no for now.

Reply:
thank you! will merge as-is